### PR TITLE
JavaJavaJavaBills ya'll

### PR DESCRIPTION
**Pull request by Meeeeeesh**
React.js

### CODE REVIEW

**Comment by Meeeeeesh** [comment]
- [ ] testing123

File: app/assets/javascripts/App.react.js [hunk 1]
```diff
@@ -1,13 +1,60 @@
...
-        <h1>Bonjour!</h1>
+        <userInfoForm create={this._createUserForm} />
+        <userList create={this.state.items} />
       </div>
     )
+  },
+
+  _fetchApp: function() {
+    request
+      .get('/app')
```
Comment (seanmpage) [inline app/assets/javascripts/App.react.js]: /app i think should be App

Comment (Meeeeeesh) [inline app/assets/javascripts/App.react.js]: that failed as well. I think it should be user_infos

Comment (Meeeeeesh) [inline app/assets/javascripts/App.react.js]: I was right. it works now.

**Pull request merged**